index.js

```diff
@@ -500,45 +517,24 @@ RingPop.prototype.lookupN = function lookupN(key, n) {
 };
 
 RingPop.prototype.reload = function reload(file, callback) {
```

Comment:
Agree with @thanodnl. I'd also check with @jwolski that it's OK to deprecate. I.e. how confident are we this is not in use?